pandas/core/frame.py

```diff
@@ -2171,6 +2174,23 @@ def from_records(
         2      1     c
         3      0     d
         """
+        if isinstance(data, DataFrame):
+            warnings.warn(
+                "Passing a DataFrame to DataFrame.from_records is deprecated. Use "
+                "set_index and/or drop instead to modify the DataFrame.",
```

Comment:
"instead"?

Reply:
Why I am using instead or placement of instead within the sentence?

Reply:
im suggesting adding "instead" at the end of the last sentence

Reply:
changed